features/networking/multus.feature

```diff
@@ -446,7 +446,7 @@ Feature: Multus-CNI related scenarios
     Then the step should succeed
     And the output should contain 2 times:
       | 1 PVID   |
-      | untagged |
+      | ntagged |
```

Comment:
I kept it like this. At this point this will work. Its not good to play with changing this definition as 1000's of cases are using it.
https://github.com/anuragthehatter/verification-tests/blob/master/features/step_definitions/common.rb#L72